src/hoax_code.erl

```diff
@@ -26,13 +26,62 @@ get_callback_list(Behaviour, ModuleName) ->
         error({not_a_behaviour, Behaviour}),
     Behaviour:behaviour_info(callbacks).
 
+backup_coverdata(ModName) ->
+    case cover:is_compiled(ModName) of
+        false ->
+            ok;
+        _Other ->
+            FileName = coverdata_filename(ModName),
+            cover:export(FileName, ModName)
+    end.
+
+coverdata_filename(ModName) ->
+    %% This assumes we start and stop in the same directory.
+    {ok, D} = file:get_cwd(),
+    filename:join(D, atom_to_list(ModName) ++ ".coverdata").
+
+restore_coverdata(ModName) ->
+    File = coverdata_filename(ModName),
+    case cover:import(File) of
+        {error, {cant_open_file, File, enoent}} ->
+            ok;
+        ok ->
+            file:delete(File),
+            ok;
+        Other ->
+            Other
+    end.
+
+has_coverdata_backup(ModName) ->
+    Filename = coverdata_filename(ModName),
+    case file:open(Filename, [read]) of
```

Comment:
Nice! This was just a case of not knowing that function was around.  I swear that I've looked for that function before and couldn't find it.  I think there might be a few places where we can move to `filelib:is_file/1` in other projects.